gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -593,11 +617,19 @@ export class AdvancedDynamicTexture extends DynamicTexture {
             if (scene!.isPointerCaptured((<PointerEvent>(pi.event)).pointerId)) {
                 return;
             }
+            //check for focused control and call the onClipboardPointerEvents
+            if (this._focusedControl) {
+                if (pi.type === PointerEventTypes.POINTERDOUBLETAP) {
+                    this._focusedControl.processKeyboard(pi.event);

```

Comment:
This looks weird to me.. I would have thought more about the input text registering to double tap observable instead of having th ADT forcefully calling a fake keyboard event